apps/indexer-coordinator/src/core/docker.service.ts

```diff
@@ -102,4 +129,32 @@ export class DockerService {
       });
     });
   }
+
+  validateFilePath(path: string): boolean {
```

Comment:
throw error when failed to pass the validation?
we can do it here or the upper level.